src/sections/Article/index.tsx

```diff
@@ -77,7 +80,8 @@ const Article = ({
           relativePath={relativePath}
           editPath={editPath}
         />
-        <ArticleComponents.Pagination previous={previous} next={next} />
+        {extraLinks}
+        <Pagination previous={previous} next={next} />
```

Comment:
Why this change? It should still be imported as `<ArticleComponents.Pagination` 🤔

Reply:
Bad merge on my part, apologies. Should be cleaned up now!